cybox/test/__init__.py

```diff
@@ -52,8 +54,8 @@ def round_trip(o, output=False, list_=False):
 
     klass = o.__class__
     if output:
-        print "Class: ", klass
-        print "-" * 40
+        print("Class: ", klass)
```

Comment:
Not needed for this PR, but I'm wondering if we should switch to using `logging` calls over `print` statements.